apps/client/src/app/components/admin-market-data/admin-market-data.component.ts

```diff
@@ -186,6 +188,18 @@ export class AdminMarketDataComponent implements OnDestroy, OnInit {
       .subscribe(() => {});
   }
 
+  public onSetBenchmark(benchmark: UniqueAsset) {
+    this.dataService
+      .postBenchmark(benchmark)
+      .pipe(takeUntil(this.unsubscribeSubject))
+      .subscribe((benchmarkProfile) => {
+        this.benchmarks.push(benchmarkProfile);
```

Comment:
I would rather reload the `/info` endpoint than having all this logic in the client.